www/css/dark.css

```diff
@@ -14,25 +15,21 @@ body {
   background-color: var(--bg-color);
 }
 
-.blocklyWorkspace{
+.blocklyWorkspace {
   color: var(--txt-color);
   background-color: var(--bg-color);
 }
 
-.blocklyToolboxDiv{
+.blocklyToolboxDiv {
   color: var(--txt-color);
   background-color: var(--bg-color);
 }
 
```

Comment:
I don't know, it just seems like there's something about the blocks that makes them more... pale.

I know it's an optical illusion, but could you make the blocks look more... alive on the dark theme?

It may just be me, and there might be no actual difference between the blocks on dark theme and the blocks on light theme :joy:

![image](https://user-images.githubusercontent.com/32238504/62171916-4d209700-b339-11e9-858c-04fb9cde747f.png)

Reply:
the blocks are not changed at all

Reply:
I know that, but the contrast between them and the dark background is different than the contrast between them and the light background. 

In my opinion they look more pale on the dark background (I may just be seeing things.)